src/test/java/org/elasticsearch/test/rest/ElasticsearchRestTestCase.java

```diff
@@ -302,4 +321,86 @@ public void test() throws IOException {
             executableSection.execute(restTestExecutionContext);
         }
     }
+
+    // don't look any further: NO TOUCHY!
+    
+    public static class Rest0Tests extends ElasticsearchRestTestCase {
```

Comment:
@javanna I think the reporting problem is fixed in the pom.xml file now with this change https://github.com/elastic/elasticsearch/pull/10656/files#diff-600376dffeb79835ede4a0b285078036R579